packages/terra-functional-testing/lib/terra-dev-site/doc/asset-server-service/AssetServerService.doc.mdx

```diff
@@ -0,0 +1,199 @@
+# Asset Server Service
+
+The asset server service initializes a server to serve compiled assets to the testing environment.
+
+## Installation
+
+The asset server service should be installed as a development dependency.
+
+```bash
+npm install --save-dev @cerner/terra-functional-testing
+```
+
+## Configuration
+
+To use the service it must be added to the services array in the `wdio.conf.js` file.
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+            // Asset server service options here.
+            // ...
+        }]
+    ],
+    // ...
+};
+```
+
+## Options
+
+### config
+
+A path to a webpack configuration file.
+
+Note: The `site` configuration option will take precedence over `config` if both are provided.
+
+Type: `string`
+
+Required: `false`
+
+Default: The asset server service will check the current working directory for a `webpack.config.js` file.
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+          config: './path/webpack.config.js'
+        }]
+    ],
+    // ...
+};
+```
+
+### index
```

Comment:
Do we need to support this option? Do we have any evidence of anyone no using index for this?

Reply:
I carried this forward from the existing implementation, I'm not aware of anyone using it today

Reply:
Removed index as a configurable option, we can always add this passively back in if necessary:

https://github.com/cerner/terra-toolkit/pull/417/commits/4d8bce55c326b1e1d67889270a32b925702ca29d